MODULE.bazel.lock

```diff
@@ -1,6 +1,6 @@
 {
   "lockFileVersion": 3,
-  "moduleFileHash": "836f0a7d2276ed93403f104a10008b94ec7e7f81b8d6921cea287f0a6d364efa",
+  "moduleFileHash": "9e42fa15fdae2a59a51ea764fd71c5e9f785b0fcb894b4e6007c56956b0082af",
```

Comment:
This happened automatically, not sure why. @fzakaria any idea why this might have happened? Could it be because I changed `BUILD.bazel` ?

Reply:
> The moduleFileHash represents the hash of the MODULE.bazel file contents. If any changes occur in this file, the hash value differs.

I think it might have been someone else change they didn't commit.
I think from the license change to MODULE.bazel -- you have to run `bazel build` to generate the lock file.

Reply:
Should this have failed bazel CI?

Reply:
Sent #2060 to address this.